README.md

```diff
@@ -182,6 +183,10 @@ https://github.com/eosrei/twemoji-color-font/releases/download/v12.0.1/TwitterCo
 *Reiterating: Only FireFox and Edge support the SVGinOT color emoji for now. Chrome will use the
 fallback black and white emoji.*
 
+## Uninstalling
+
+There are uninstall scripts for [Windows](https://github.com/eosrei/twemoji-color-font/blob/master/windows/uninstall.cmd) and [Linux](https://github.com/eosrei/twemoji-color-font/blob/master/linux/uninstall.sh) available. 
```

Comment:
Thanks for your contribution! We should call out the release files contain these uninstall scripts and they don't need to be downloaded separately.

Please:
* Use Markdown reference links instead of in-line links: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links
* Use a relative path: https://help.github.com/en/articles/about-readmes#relative-links-and-image-paths-in-readme-files
* Wrap the line at 80 characters, if needed, to match the rest of the document.

Reply:
Done!

Feel free to squash the commits and set the appropriate commit message :relaxed: